askama_derive/src/generator.rs

```diff
@@ -13,6 +13,27 @@ use parser::node::{
 use parser::{Expr, Node};
 use quote::quote;
 
+const _ASKAMA_: &str = "::askama::";
+const _PREFIX: &str = if cfg!(feature = "with-actix-web") {
+    "::askama_actix"
+} else if cfg!(feature = "with-axum") {
+    "::askama_axum"
+} else if cfg!(feature = "with-gotham") {
+    "::askama_gotham"
+} else if cfg!(feature = "with-hyper") {
+    "::askama_hyper"
+} else if cfg!(feature = "with-mendes") {
+    "::askama_mendes"
+} else if cfg!(feature = "with-rocket") {
+    "::askama_rocket"
+} else if cfg!(feature = "with-tide") {
+    "::askama_tide"
+} else if cfg!(feature = "with-warp") {
+    "::askama_warp"
+} else {
+    ""
```

Comment:
Why shouldn't this be `::askama`? If this is acting as a sentinel value, I'd prefer to make this an `Option` instead.

Reply:
This (`const _PREFIX: &str`) is the integration prefix. It's "concatenated" with `const _ASKAMA_: &str` i.e. `"::askama::"`, so it needs to remain empty when no integration is present, otherwise the result would be `::askama::askama::`. It's not a sentinel value AFAICT.

Reply:
Ahh, yeah, that's fair.